lib/streamlit/DeltaGenerator.py

```diff
@@ -1740,7 +1740,8 @@ def slider(
         format=None,
         key=None,
     ):
-        """Display a slider widget.
+        """Display a slider widget. Integers, floats, and tuples/lists of
+        integers are supported.
```

Comment:
This needs to fit in a single line. Also, the second sentence isn't adding much.

A clearer doc would be:

```py
"""Display a slider widget.

This also allows you to draw a range slider, by passing a two-element tuple or list as the `value`.
```

Reply:
And then we could add an example in the examples section below too.

Reply:
Fixing in the next push.